libs/core/src/lib/utils/functions/key-util.ts

```diff
@@ -36,6 +38,7 @@ const keyMap: Map<number, string[]> = new Map([
     [ESCAPE, ['Escape', 'Esc']],
     [DELETE, ['Delete', 'Del']],
     [ENTER, ['Enter']],
+    [MAC_ENTER, ['Return']],
```

Comment:
wouldn't it be reasonable to combine these 2? I'm not sure it will be ever a case when we need to track these 2 keys apart

Reply:
Removed MAC_ENTER